internal/http/middleware/auth.go

```diff
@@ -46,6 +70,57 @@ func (m *AuthMiddleware) OnRequest(deps model.Dependencies, c model.WebContext)
 	return nil
 }
 
+func (m *AuthMiddleware) ssoAccount(deps model.Dependencies, c model.WebContext) *model.AccountDTO {
```

Comment:
I would use `getSSOAccount(...) (*model.AccountDTO, error)` instead of just logging internally, so we can log all the errors directly in the parent function. What do you think?